deis-builder.service failing to start on brand new EC2 CloudFormation Stack

I've recently attempted to launch a three node EC2 cluster for deis, but one of the services is failing to start.

``` bash
15:49 deis‹master› » fleetctl --version
fleetctl version 0.2.0
```

```
15:51 deis‹master› » git rev-parse HEAD
4c2dfe6a18dd89070c614f802eefc2af1cde2571
```

```
15:52 deis‹master› » git diff
diff --git a/contrib/coreos/user-data b/contrib/coreos/user-data
index f4b3bd8..44696c6 100644
--- a/contrib/coreos/user-data
+++ b/contrib/coreos/user-data
@@ -4,7 +4,7 @@ coreos:
   etcd:
     # generate a new token for each unique cluster from https://discovery.etcd.io/new
     # uncomment the following line and replace it with your discovery URL
-    # discovery: https://discovery.etcd.io/12345693838asdfasfadf13939923
+    discovery: https://discovery.etcd.io/a-unique-id
     addr: $private_ipv4:4001
     peer-addr: $private_ipv4:7001
   units:
diff --git a/contrib/ec2/cloudformation.json b/contrib/ec2/cloudformation.json
index 9a90c28..7dbd55a 100644
--- a/contrib/ec2/cloudformation.json
+++ b/contrib/ec2/cloudformation.json
@@ -1,6 +1,6 @@
 [
-    {
-        "ParameterKey":     "KeyPair",
-        "ParameterValue":   "deis"
-    }
+  {
+    "ParameterKey":     "KeyPair",
+    "ParameterValue":   "deis"
+  }
 ]
```

```
15:07 deis‹master› » make run
Warning: Permanently added 'ec2-54-86-97-191.compute-1.amazonaws.com,54.86.97.191' (RSA) to the list of known hosts.
fleetctl --strict-host-key-checking=false submit  registry/systemd/deis-registry.service  logger/systemd/deis-logger.service  cache/systemd/deis-cache.service  database/systemd/deis-database.service  router/systemd/deis-router.service
Starting Deis! Deis will be functional once all services are reported as running... 
fleetctl --strict-host-key-checking=false start  registry/systemd/deis-registry.service  logger/systemd/deis-logger.service  cache/systemd/deis-cache.service  database/systemd/deis-database.service  router/systemd/deis-router.service
Job deis-registry.service scheduled to 9f211f33.../172.31.41.50
Job deis-logger.service scheduled to 5bf75f02.../172.31.23.89
Job deis-cache.service scheduled to 9f211f33.../172.31.41.50
Job deis-database.service scheduled to 254435bd.../172.31.4.66
Job deis-router.service scheduled to 9f211f33.../172.31.41.50
Waiting for deis-registry to start (this can take some time)... 
Done! Waiting for deis-controller...
fleetctl --strict-host-key-checking=false submit controller/systemd/*
fleetctl --strict-host-key-checking=false start controller/systemd/*
Job deis-controller.service scheduled to 5bf75f02.../172.31.23.89
Done! Waiting for deis-builder to start (this can also take some time)... 
fleetctl --strict-host-key-checking=false submit builder/systemd/*
fleetctl --strict-host-key-checking=false start builder/systemd/*
Job deis-builder.service scheduled to 5bf75f02.../172.31.23.89
One or more services failed! Check which services by running 'make status'
You can get detailed output with 'fleetctl status deis-servicename.service'
This usually indicates an error with Deis - please open an issue on GitHub or ask for help in IRC
make: *** [start] Error 1
```

```
15:37 deis‹master› » make status
fleetctl --strict-host-key-checking=false list-units
UNIT            LOAD    ACTIVE  SUB DESC        MACHINE
deis-builder.service    loaded  failed  failed  deis-builder    5bf75f02.../172.31.23.89
deis-cache.service  loaded  active  running deis-cache  9f211f33.../172.31.41.50
deis-controller.service loaded  active  running deis-controller 5bf75f02.../172.31.23.89
deis-database.service   loaded  active  running deis-database   254435bd.../172.31.4.66
deis-logger.service loaded  active  running deis-logger 5bf75f02.../172.31.23.89
deis-registry.service   loaded  active  running deis-registry   9f211f33.../172.31.41.50
deis-router.service loaded  active  running deis-router 9f211f33.../172.31.41.50
```

```
15:42 deis‹master› » fleetctl journal deis-builder.service
-- Logs begin at Sat 2014-05-03 19:02:20 UTC, end at Sat 2014-05-03 19:17:58 UTC. --
May 03 19:17:54 ip-172-31-23-89 sh[5191]: /bin/sh: connect: Connection refused
May 03 19:17:54 ip-172-31-23-89 sh[5191]: /bin/sh: /dev/tcp/172.31.23.89/2222: Connection refused
May 03 19:17:55 ip-172-31-23-89 sh[5191]: /bin/sh: connect: Connection refused
May 03 19:17:55 ip-172-31-23-89 sh[5191]: /bin/sh: /dev/tcp/172.31.23.89/2222: Connection refused
May 03 19:17:56 ip-172-31-23-89 sh[5191]: /bin/sh: connect: Connection refused
May 03 19:17:56 ip-172-31-23-89 sh[5191]: /bin/sh: /dev/tcp/172.31.23.89/2222: Connection refused
May 03 19:17:57 ip-172-31-23-89 sh[5191]: /bin/sh: connect: Connection refused
May 03 19:17:57 ip-172-31-23-89 sh[5191]: /bin/sh: /dev/tcp/172.31.23.89/2222: Connection refused
May 03 19:17:58 ip-172-31-23-89 sh[5191]: /bin/sh: connect: Connection refused
May 03 19:17:58 ip-172-31-23-89 sh[5191]: /bin/sh: /dev/tcp/172.31.23.89/2222: Connection refused
```

I pinged the IRC room, but didn't get any love.

Any thoughts on what I could be doing wrong?

--- a/contrib/ec2/deis.template
+++ b/contrib/ec2/deis.template
@@ -159,8 +159,13 @@
         "InstanceType": {"Ref": "InstanceType"},
         "KeyName": {"Ref": "KeyPair"},
         "SecurityGroups": [{"Ref": "CoreOSSecurityGroup"}, {"Ref": "DeisSecurityGroup"}],
-        "UserData" : { "Fn::Base64": { "Fn::Join": [ "", [ ] ] }
-        }
+        "UserData" : { "Fn::Base64": { "Fn::Join": [ "", [ ] ] } },
+        "BlockDeviceMappings" : [
+          {
+            "DeviceName" : "/dev/sda",
+            "Ebs" : { "VolumeSize" : "100" }
+          }
+        ]
       }
     }
   }

